transport-native-io_uring/src/test/java/io/netty/channel/uring/PollRemoveTest.java

```diff
@@ -81,7 +81,7 @@ public void initChannel(SocketChannel ch) throws Exception {
     public void test() throws Exception {
 
         io_uring_test();
-
+        Thread.sleep(1000);
         System.out.println("io_uring --------------------------------");
```

Comment:
Can we use a Logger instead of System.out.println(Object)?

Reply:
yes... I will do a full cleanup now so check style will pass etc.